norddrop/src/ffi/mod.rs

```diff
@@ -77,8 +77,11 @@ pub struct norddrop(Mutex<NordDropFFI>);
 ///   }
 /// ]
 /// ```
+///
+/// # Safety
+/// The pointers provided must be valid
```

Comment:
Since these will modify the resulting `norddrop.h` file, could you also include the modified bindings as well?

Reply:
Sure

Reply:
Done